code/modules/research/server.dm

```diff
@@ -47,8 +47,6 @@
 	name += " [uppertext(num2hex(server_id, -1))]" //gives us a random four-digit hex number as part of the name. Y'know, for fluff.
 	SSresearch.servers |= src
 	stored_research = SSresearch.science_tech
-	var/obj/item/circuitboard/machine/B = new /obj/item/circuitboard/machine/rdserver(null)
-	B.apply_default_parts(src)
```

Comment:
what was this all about?

Reply:
`apply_default_parts` should only be called in machine/circuitboard code. Processors and Servers shouldnt need to call it in their own inits